test-sbt/js/src/main/scala/zio/test/sbt/ZTestRunner.scala

```diff
@@ -98,34 +98,24 @@ sealed class ZTestTask(
     spec match {
       case NewSpecWrapper(zioSpec) => {
 
-        val argslayer: ULayer[ZIOAppArgs] =
-          ZLayer.succeed(
-            ZIOAppArgs(Chunk.empty)
-          )
-
-        val filledTestlayer: ZLayer[Scope, Nothing, TestEnvironment] =
-          zio.ZEnv.live >>> TestEnvironment.live
-
-        val layer: ZLayer[Scope, Error, zioSpec.Environment] =
-          (argslayer +!+ filledTestlayer) >>> zioSpec.layer.mapError(e => new Error(e.toString))
-
         val fullLayer: Layer[
           Error,
-          zioSpec.Environment with ZIOAppArgs with TestEnvironment with zio.Console with System with Random with Clock with Scope
+          zioSpec.Environment with ZIOAppArgs with TestEnvironment with Scope with TestLogger
         ] =
-          Scope.default >>> (layer +!+ argslayer +!+ filledTestlayer +!+ ZLayer.environment[Scope])
+          constructLayer[zioSpec.Environment](zioSpec.layer)
 
-        val testLoggers: Layer[Nothing, TestLogger] = sbtTestLayer(loggers)
         Runtime(ZEnvironment.empty, zioSpec.hook(zioSpec.runtime.runtimeConfig)).unsafeRunAsyncWith {
           val logic =
             for {
-              spec <- zioSpec
-                        .runSpec(FilteredSpec(zioSpec.spec, args), args, sendSummary)
-                        .provideLayer(
-                          testLoggers +!+ fullLayer
-                        )
-              events = ZTestEvent.from(spec, taskDef.fullyQualifiedName(), taskDef.fingerprint())
-              _     <- ZIO.foreach(events)(e => ZIO.attempt(eventHandler.handle(e)))
+              _ <- zioSpec
+                     .runSpec(FilteredSpec(zioSpec.spec, args), args)
+                     .provideLayer(
+                       fullLayer
+                     )
+              // TODO Confirm if/how these events needs to be handled
```

Comment:
Probably better to open an issue for this.

Reply:
Updated comment to reference https://github.com/zio/zio/issues/6481